django/core/management/commands/loaddata.py

```diff
@@ -249,15 +304,26 @@ def load_label(self, fixture_label):
                     handle_forward_references=True,
                 )
 
-                for obj in objects:
-                    objects_in_fixture += 1
-                    if self.save_obj(obj):
-                        loaded_objects_in_fixture += 1
-                        if show_progress:
-                            self.stdout.write(
-                                "\rProcessed %i object(s)." % loaded_objects_in_fixture,
-                                ending="",
-                            )
+                if self.bulk_create:
+                    counters = self.bulk_create_obj(objects)
+                    objects_in_fixture += counters["objects_in_fixture"]
+                    loaded_objects_in_fixture += counters["loaded_objects_in_fixture"]
+                    if show_progress:
+                        self.stdout.write(
+                            "\rProcessed %i object(s)." % loaded_objects_in_fixture,
+                            ending="",
+                        )
```

Comment:
I agree, but the stdout/stderr is not tested anywhere in this module  (verbosity is always set to 0 for all tests). To properly test stdout/stderr, I generally capture stdout and stderr on the setUp() of the class, and then evaluate what was actually sent (instead of merely setting verbosoty to 3 and running the happy path without checking what is actually written). I have no problem adding code for this, but it makes the PR quite a bit larger and it turns into a generic change rather than a small increment. Kindly advice what you would prefer.